README.md

```diff
@@ -84,29 +84,38 @@ Starts the interactive setup wizard that will guide you through the available op
 pnpm start
 ```
 
-### `pnpm push`
+### `pnpm deploy`
 
-Updates the remote Saleor instance according to the local configuration.
+Deploys the local configuration to the remote Saleor instance with mandatory diff preview and safety confirmations.
 
 ```bash
-# Basic usage
-pnpm push --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token"
+# Basic usage with diff preview and confirmation
+pnpm deploy --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token"
 
 # With custom config file
-pnpm push --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --config="production.yml"
+pnpm deploy --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --config="production.yml"
+
+# CI mode (skip confirmations for automated environments)
+pnpm deploy --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --ci
+
+# Force mode (skip all confirmations - use with extreme caution)
+pnpm deploy --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --force
 
-# Quiet mode (suppress output)
-pnpm push --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --quiet
+# Skip diff preview (not recommended)
+pnpm deploy --url="https://your-store.saleor.cloud/graphql/" --token="your-app-token" --skip-diff
 
 # Show help
-pnpm push --help
+pnpm deploy --help
 ```
 
 **Arguments:**
 
 - `--url` (required): Saleor instance URL
 - `--token` (required): Saleor API token
 - `--config` (optional): Configuration file path (default: `config.yml`)
+- `--ci` (optional): CI mode - skip confirmations for automated environments
```

Comment:
question: what are the use cases for having all three of these? I think we would be good by only having `force`